index.html

```diff
@@ -190,6 +190,42 @@ <h3 class="Skill">Skills</h3>
             </div>
 
         </section>
+        <footer>
+            <section class="form">
+                <p class="para1">
+            I'm always<br> interested in hearing <br> about new projects, <br> so if you'd like to <br>chat please get in
+                </p>
+                <form action="/action_page.php" class="form_div">
+                    <div class="fname"> 
+                    <input maxlength="30" type="text" id="name" name="user_name" placeholder="Full name" class="full_name" required>
+                    <hr class="name_line">
+                    </div>
+                    <div class="email">
+                        <input type="email" id="mail" name="user_email" placeholder="Email address" class="email_address" required>
+                        <hr class="email_line">
+                    </div>
+                    <div class="text_area">
+                        <textarea maxlength="500" id="msg" name="user_message" placeholder="Enter text here" class="text_box" required></textarea>
+                    </div>
+                    <button type="submit"  class="get_in_touch">
+                        Get in touch
+                    </button>
+                </form>
+            </section>
+            <section class="last_game">
+                <div class="last-icons">
+                    <hr class="icons_line">
+                    <ul>
+                        <li><a href="#"><img src="images/Github-black.png" alt="github-icon"></a> </li>
+                        <li><a href="#"><img src="images/linkedin-black.png" alt="linked-icon"></a> </li>
+                        <li><a href="#"><img src="images/chill-black.png" alt="chill-icon"></a> </li>
+                        <li><a href="#"><img src="images/twitter-black.png" alt="twitter-icon"></a> </li>
+                        <li><a href="#"><img src="images/Medium-black.png" alt="Medium-icon"></a> </li>
+                    </ul>
+                    <hr class="final_line">
```

Comment:
- [ ] **\[OPTIONAL\]** This black bar is not part of the template, it's just there. So, you can consider removing it.
![image](https://user-images.githubusercontent.com/92215878/187491297-7d30d6c4-0581-41e0-bc8c-3a79cf39de75.png)